### ISSUE TITLE
ynet.co.il: breakage

### ISSUE THREAD

**Issue by sha-265** [uBlockOrigin/uAssets#22044]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.ynet.co.il/home/0,7340,L-8,00.html`

### Category

nuisance

### Description

When loading a page in this website in FF for Android with uBO enabled, there is a blank page for few seconds, and only after that the full page is loaded. 

That's not happening with uBO Lite.

### Other extensions used

https://github.com/uBlockOrigin/uAssets/assets/4103710/d6582756-67c9-4789-ad54-06ead2794361

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screenshot_20240118-010225](https://github.com/uBlockOrigin/uAssets/assets/4103710/e29b705e-d39b-4a7e-927d-bde046641720)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.55.0
Firefox Mobile: 121
filterset (summary):
 network: 115195
 cosmetic: 36788
 scriptlet: 18784
 html: 2023
listset (total-discarded, last-updated):
 removed:
  plowe-0: null
 added:
  fanboy-thirdparty_social: 68-1, now
  ISR-0: 1779-5, now
 default:
  user-filters: 1-0, never
  ublock-filters: 37412-359, now
  ublock-badware: 7740-11, now
  ublock-privacy: 969-1, now
  ublock-unbreak: 2239-2, now
  ublock-quick-fixes: 230-0, now
  adguard-mobile: 9049-49, now
  easylist: 77497-16340, now
  easyprivacy: 43452-47, now
  urlhaus-1: 9255-0, now
filterset (user): [array of 1 redacted]
trustedset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 832 ms (selfie)
 maxAssetCacheWait: 166 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 23
 network:
  ynet.co.il: 6
  brandmetrics.com: 1
  browsiprod.com: 2
  chartbeat.com: 1
  cloudflareinsights.com: 2
  doubleclick.net: 1
  dxmdp.com: 2
  google.com: 2
  googletagmanager.com: 1
  infogram.com: 1
  taboola.com: 4
 extended:
  ##+js(abort-on-property-write, upManager)
  ##+js(no-setTimeout-if, offsetHeight)
```
</details>

**Comment by D4niloMR** [comment]
It's proposital.

Test: `ynet.co.il##style#yitwall:remove()`


```
<script>
const TIMEOUT_DURATION = 10000;

fetch("https://ads.google.com", {
    method: "HEAD",
    mode: "no-cors",
    cache: "no-store",
}).catch(() => {
    const x = document.getElementById("yitwall");
    x && x.parentElement.removeChild(x);
});

const intervalCheckAd = setInterval(() => {
        if (window && window.googletag && window.googletag.apiReady) {
            googletag.pubads().addEventListener('slotRenderEnded', function() {
                setTimeout(() => {
                    const x = document.getElementById("yitwall");
                    x && x.parentElement.removeChild(x);
                    window.clearInterval(intervalCheckAd);
                }, 2000)
            })
        }
    }, 100)

setTimeout(() => {
    window.clearInterval(intervalCheckAd);
    const x = document.getElementById("yitwall");
    x && x.parentElement.removeChild(x);
}, TIMEOUT_DURATION)
</script>
```

**Comment by mapx-** [comment]
continue here: https://github.com/easylist/EasyListHebrew/issues/391

**Comment by sha-265** [comment]
> `ynet.co.il##^style#yitwall`

@D4niloMR, this is working for me.

**Comment by mapx-** [comment]
to be added in the hebrew list.

**Comment by rerere123123** [comment]
This issue is back again on iOS.
For about ~10 seconds after loading any page on this domain, most of the screen is blank (completely white) and scrolling doesn't work. A few seconds later, the whole website appears and scrolling works.
*This issue exists on iOS 26 and on iOS 27 beta.
*This issue has started since updating to version 2026.804.1652. The update released today (2026.811.1529) does not fix it.
*Website works fine without uBlock Origin Lite.

<details>

<img width="1206" height="2622" alt="Image" src="https://github.com/user-attachments/assets/cae31332-92bb-48a7-a957-211e0a2cff44" />

</details>

**Comment by rerere123123** [comment]
Looks like version 2026.812.1211 fixed it.
